### PR TITLE
Add/proptype card home

### PR DESCRIPTION
**Pull request by Afoucheaux**
## Purpose
Add PropType on the card and home components. Refactor css to clean up code.
## Approach
Turing lesson and React Doc's
#### Open Questions and Pre-Merge TODOs
- [x] Ensure functionality did not change.
- [x] Review code for any refactoring opportunity that may have been missed. 
## Learning
React PropType

### CODE REVIEW

File: src/MovieCard/MovieCard.css [hunk 1]
```diff
@@ -33,6 +33,11 @@
   -12px -12px 50px 1px rgba(194, 3, 252, .4);
 }
 
+.card-image:hover {
+  margin: 0% 5% 0% 5%;
+  object-fit: cover;
+}
+
```
Comment (elizhahn) [inline src/MovieCard/MovieCard.css]: I took this out on the last PR, I realized there was a hover on the card container and the image, and  I think it is too much hover action on the card and feels a bit glitchy. Looks like it's being added back in here but taken out below in the media queries

**Review by elizhahn — CHANGES_REQUESTED** [review]

File: src/MovieCard/MovieCard.css (hunk 1 above)
Comment (Afoucheaux) [reply src/MovieCard/MovieCard.css]: Ok it was on other screen sizes so I thought so was a little unsure I will pull it out and let you know when its done.

Comment (Afoucheaux) [reply src/MovieCard/MovieCard.css]: Fixed

Comment (elizhahn) [reply src/MovieCard/MovieCard.css]: Yeah I totally didn't remove it from the media queries and they should be so that makes sense.

**Review by elizhahn — APPROVED** [review]

**Pull request merged**